pages/data-providers/[data-provider-id]/compliance.jsx

```diff
@@ -40,26 +40,31 @@ const SidebarContent = ({ context: { oai, originalId, authors, title } }) => {
  *       of Layout.Main and explicitly pass props.
  *       We should get rid out of it!
  */
-const DepositDates = ({
-  className,
-  store,
-  tag: Tag = 'main',
-  ...restProps
-}) => (
+const Compliance = ({ className, store, tag: Tag = 'main', ...restProps }) => (
   <Tag className={[styles.container, className].join(' ')} {...restProps}>
-    <h1>Deposit compliance</h1>
+    <h1>REF 2021 Compliance</h1>
 
-    <Card className={styles.complianceLevel} tag="section">
+    <Card className={styles.dataOverview} tag="section">
       <h2>{texts.compliance.title}</h2>
-      {store.depositDates.complianceLevel ? (
-        <Markdown>
-          {texts.compliance.body.render({
-            amount: (100 - store.depositDates.complianceLevel).toFixed(1),
-          })}
-        </Markdown>
-      ) : (
-        <p>{texts.noData.body}</p>
-      )}
+      <div className={styles.numbers}>
+        <NumericValue
+          tag="div"
+          value={valueOrDefault(
+            store.depositDates.depositDatesCount,
+            'Loading...'
+          )}
+          caption="outputs counted"
+        />
+        <NumericValue
+          tag="div"
+          value={valueOrDefault('2.7', 'Loading...')}
+          append="%"
+          caption="non-compliant"
+        />
+      </div>
+      <Button tag="a" href="#depositDatesCard" variant="contained">
```

Comment:
I prefer URLs, IDs, classes with _kebab-case_, JS/Java names with _camelCase_ (along with Python/C++ names in _snake_case_). It's the way a certain language or domain tends to look like. We follow this everywhere. Could you please change it accordingly here?

We should standardize this properly organisation-wise.